### PR TITLE
Features/add nonserialized backing fields

### PR DESCRIPTION
**Pull request by luizfbicalho**
Closes: #1628

### CODE REVIEW

**Comment by StefanOssendorf** [comment]
Sorry I'm late to the party. I've asked a question in the related issue.

File: Source/Csla/Core/FieldManager/FieldData.cs [hunk 1]
```diff
@@ -76,6 +81,11 @@ object IFieldData.Value
           Value = (T)value;
       }
     }
+    /// <summary>
+    /// Gets a value indicating whether this field
+    /// references a serializable property.
+    /// </summary>
+    public bool IsSerializable { get; set; }
```
Comment (rockfordlhotka) [inline Source/Csla/Core/FieldManager/FieldData.cs:88]: Shouldn't this be readonly?

File: Source/Csla/Core/FieldManager/FieldDataManager.cs [hunk 2]
```diff
@@ -77,7 +77,7 @@ internal void SetParent(BusinessBase parent)
     /// </summary>
     public List<IPropertyInfo> GetRegisteredProperties()
     {
-      return [.._propertyList];
+      return [.. _propertyList];
```
Comment (rockfordlhotka) [inline Source/Csla/Core/FieldManager/FieldDataManager.cs:80]: Why the added space?

**Review by rockfordlhotka — CHANGES_REQUESTED** [review]

File: Source/Csla/Core/FieldManager/FieldDataManager.cs (hunk 2 above)
Comment (luizfbicalho) [reply Source/Csla/Core/FieldManager/FieldDataManager.cs:80]: was added by the code cleanup

File: Source/Csla/Core/FieldManager/FieldData.cs (hunk 1 above)
Comment (luizfbicalho) [reply Source/Csla/Core/FieldManager/FieldData.cs:88]: added a private set

**Review by rockfordlhotka — APPROVED** [review]

**Pull request merged**